src/jsdoc_transformer.ts

```diff
@@ -0,0 +1,937 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {addSyntheticTrailingComment} from 'typescript';
+
+import {hasExportingDecorator} from './decorators';
+import * as googmodule from './googmodule';
+import * as jsdoc from './jsdoc';
+import {ModuleTypeTranslator} from './module_type_translator';
+import * as transformerUtil from './transformer_util';
+import * as ts from './typescript';
+
+export interface AnnotatorHost {
```

Comment:
Doc. I think here 'Annotator' is the tsickle term for "thing that adds closure comments", right?

Reply:
Done (though only moved, in my defense!)